release_notes/RELEASE.txt

```diff
@@ -38,6 +38,33 @@ New features and changes
 ========================
     Configuration:
     -------------
+    - Tools are now built by default in CMake
+
+      The option to build and test the HDF4 tools hdiff, hrepack, hdfimport,
+      and dumper was disabled by default. It is now enabled by default.
+
+      This option also controls the ability to build the netCDF tools.
+
+      (DER - 2023/02/28)
+
+    - Added an option to disable building the netCDF tools ncdump and ncgen
+
+      HDF4 builds netCDF 2.3.2 versions of ncdump and ncgen. These can conflict
+      with the official versions so an option to disable building them has
+      been added.
+
+        CMake:        HDF_ENABLE_NETCDF_TOOLS
+        Autotools:    --enable-netcdf-tools
+
+      This option can be set independently of the option that controls netCDF
+      name mangling (HDF_ENABLE_NETCDF / --enable-netcdf) as no C library
+      functionality is affected by that option.
+
+      This option is enabled by default. HDF4_BUILD_TOOLS must be enabled to
+      build the netCDF tools.
```

Comment:
HDF4_BUILD_TOOLS is required for CMake.  Is there a similar requirement for autotools, or is it automatic?